### DISCUSSION TITLE
How to get the old like, dislike, share, download... bar back?

### DISCUSSION THREAD

**Discussion by 0evers** [PoomSmart/YTABConfig#21] [Q&A]
I would be very thankful for any advice, also because RYD does not work properly anymore.

![image](https://user-images.githubusercontent.com/78879932/198753018-6d98f81f-9e3f-42b7-9a4c-ba750e09507c.jpeg)

![image](https://user-images.githubusercontent.com/78879932/198753047-09f925b1-db81-4613-a51e-aab0449815ec.png)

**Comment by lemonke444** [comment]
try deactivating this flags YTColdConfig.uiSystemsClientGlobalConfigEnableModernButtonsForNative: 0
YTColdConfig.uiSystemsClientGlobalConfigIosEnableSnackbarModernization: 0

i have the same issue then deactivated this two flags and the old like, dislike etc came back :D

**Comment by 0evers** [comment]
Thank you, but unfortunately these flags have not changed anything in the latest uyou+ version.

**Reply by lemonke444** [reply]
> Thank you, but unfortunately these flags have not changed anything in the latest uyou+ version.
> […](#)

deactive this flags :\

uiSystemsClientGlobalConfigUseDarkerPal etteBgColorForNative 
uiSystemsClientGlobalConfigUseDarkerPal etteTextColorForNative 
useDarkerPaletteBgColorForElements
useDarkerPaletteTextColorForElements

**Reply by 0evers** [reply]
> > Thank you, but unfortunately these flags have not changed anything in the latest uyou+ version.
> 
> > […](#)
> 
> 
> 
> deactive this flags :\
> 
> 
> 
> uiSystemsClientGlobalConfigUseDarkerPal etteBgColorForNative 
> 
> uiSystemsClientGlobalConfigUseDarkerPal etteTextColorForNative 
> 
> useDarkerPaletteBgColorForElements
> 
> useDarkerPaletteTextColorForElements 

Thanks, but these flags have not changed anything either.

**Reply by lemonke444** [reply]
i found a solution just downgrade the app. im using youtube version  16.42.3 and has the old interface :D
